include/deal.II/numerics/fe_field_function.templates.h

```diff
@@ -86,7 +86,8 @@ namespace Functions
     FEValues<dim> fe_v(mapping, cell->get_fe(), quad,
                        update_values);
     fe_v.reinit(cell);
-    std::vector< Vector<double> > vvalues (1, values);
+    std::vector< Vector<typename VECTOR::value_type> > vvalues (1);
+    vvalues[0].reinit(values.size());
```

Comment:
I find it easier to read (because it's a straight initialization) if you write it as
```
std::vector< Vector<typename VECTOR::value_type> > 
  vvalues (1, Vector<typename VECTOR::value_type>(values.size()));
```
That's a bit of a long line, but at least you're not spreading the initialization out over two lines, making the reader wonder about what's happening in the second line.